src/db.py

```diff
@@ -92,13 +87,14 @@ def add_student_to_course(cid, new_student_id):
 def remove_student_from_course(cid, student_id):
 	return mongo.db.courses.update({"_id": cid}, {"$pull": {"student_ids": student_id}})
 
-
+# change this func
 def add_admin_to_course(cid, staff_id):
-	return mongo.db.courses.update({"_id": cid}, {"$addToSet": {"admin_ids": staff_id}})
-
+	# the way an admin should be added to the course
+	return mongo.db.courses.update({"_id": cid}, {'$set' : {f"staff.{staff_id}": {"is_admin": True}}})
 
+# change this func
 def remove_admin_from_course(cid, staff_id):
-	return mongo.db.courses.update({"_id": cid}, {"$pull": {"admin_ids": staff_id}})
+	return mongo.db.courses.update({"_id": cid}, {'$set' : {f"staff.{staff_id}": {"is_admin": False}}})
```

Comment:
Both of these functions assume that the staff dict only contain `is_admin` one field. This is the case now, but in case we add more fields, this won't be very extendable. Maybe change it to something like
```
{"$set" : {f"staff.{staff_id}.is_admin": False}}
```